ci-operator/config/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10.yaml

```diff
@@ -61,6 +61,11 @@ tests:
     test:
     - ref: csi-driver-shared-resource-e2e
     workflow: csi-driver-shared-resource-e2e-techpreview
+- as: e2e-aws-techpreview
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
@gabemontero AFAIU adding `skip_if_only_changed` at one place (presubmits file) i.e https://github.com/openshift/release/pull/24612/files#diff-b6158c401a6d08e8c4ae2868c1a04f505b7fcbe86c21b46cb587c8310293fa93R95 should be enough, no? Correct me if I am wrong.